lib/mutant/reporter/cli/printer.rb

```diff
@@ -3,10 +3,34 @@ class Reporter
     class CLI
       # CLI runner status printer base class
       class Printer
-        include AbstractType, Delegator, Adamantium::Flat, Concord.new(:output, :object), Procto.call(:run)
+        include AbstractType,
+                Adamantium::Flat,
+                Concord.new(:output, :object),
+                Procto.call(:run)
 
         private_class_method :new
 
+        # Create delegators to object
+        #
+        # @return [undefined]
+        def self.delegate(*names)
```

Comment:
Oh I see. It's delegating to the `object`. You *could* make this an explicit parameter to `delegate` and generalize it.